.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -39,8 +39,21 @@ steps:
 
   - wait: ~
 
-  - label: ":judge::seedling: Trigger Manual Tests Phase"
-    command: "make -C /agent trigger-manual-verification-phase"
-    if: build.branch == "main"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/manual-verification-agent:0.0.2"
+  - group: "Kibana Release Manager"
+    steps:
+      - label: ":judge::seedling: Trigger Manual Tests Phase - Kibana Release Manager"
+        command: "make -C /agent trigger-manual-verification-phase"
+        env:
+          USE_GROUP_LABEL: true
```

Comment:
we my want to ping a slack group here or not?

Reply:
@elastic/kibana-operations WDYT?